src/scalarstats.jl

```diff
@@ -45,6 +61,12 @@ function trimmean(x::RealArray, p::Real)
 end
 
 # compute mode, given the range of integer values
+"""
+    mode(a[, rng]) -> Int
```

Comment:
Yeah, the docs haven't been updated yet, as the discussion has stalled at https://github.com/JuliaLang/julia/pull/16477. But I think it's pretty clear that we will follow the real Julia syntax where it exists. `->`/`→` can still be used to give names to return values, but not for types.